src/components/DiscoveBanner/DiscoverBanner.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Button from '../Button/Button';
+import {fonts} from '../../globalStyles/fonts';
+
+const DiscoverBannerStyled = styled("div")`
+    ${fonts}
+    max-width: 1167px;
+    margin-inline: auto;
+    text-align: center;
+    background-color: rgba(255, 255, 255, 1);
+    transform: translateY(50%);
+
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+        padding-inline-start: 176px;
+        padding-inline-end: 140px;
+
+        h3 {
+            font-family: "Futura Std";
+            font-size: 50px;
+            line-heigth: 60px;
+            color: #041D42;
+            margin: 0;
+            margin-block-start: 10px;
+        }
+
+        p {
+            font-family: "Garamond";
+            font-size: 30px;
+            line-heigth: 33.75px;
+            color: #979797;
+            margin: 0;
+            margin-block-start: 63px;
+        }
+    }
+
+    @media (min-width: 900px) and (max-width: 1200px) {
```

Comment:
Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0